.github/workflows/generate.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: apigentools validate
 
       - name: Render upstream templates
-        run: apigentools templates -p template-patches openapi-git
+        run: apigentools templates
```

Comment:
Can you explain a little bit why this change is needed?

Reply:
[Actually this step can be completely removed ](https://github.com/arduino/clients-iot-api/pull/53/files#diff-7e093faf90eb45ac143b4e20c233deaeL35-L37), as the ver 1.0.0 of `apigentools` handles the templates download in the configuration file as you can see here:
https://github.com/arduino/clients-iot-api/pull/51/files#diff-8bbac5be38be0acd514047133a3900d7R11-R16

Reply:
If it's not needed anymore, I'd remove it

Reply:
We did it in https://github.com/arduino/clients-iot-api/pull/53